### PR TITLE
Remove languages.php and getLanguages hook

### PR DESCRIPTION
**Pull request by ausi**

### CODE REVIEW

**Review by Toflar — CHANGES_REQUESTED** [review]
Same here, UPGRADE.md

**Comment by ausi** [comment]
> Same here, UPGRADE.md

Done in ee894fbe36846ee27207df2d48fb4a2282a42a91

**Review by m-vo — DISMISSED** [review]

File: core-bundle/src/Resources/contao/library/Contao/System.php [hunk 1]
```diff
@@ -465,21 +465,6 @@ public static function loadLanguageFile($strName, $strLanguage=null, $blnNoCache
...
-			{
-				$GLOBALS['TL_LANG']['LNG'][$strLocale] = null;
-			}
-
-			foreach (self::getContainer()->get('contao.intl.locales')->getLocales($strCacheKey) as $strLocale => $strLabel)
-			{
-				$GLOBALS['TL_LANG']['LNG'][$strLocale] = $strLabel;
-			}
-		}
-
```
Comment (m-vo) [inline core-bundle/src/Resources/contao/library/Contao/System.php:482]: This currently makes the `SystemTest#testLoadsLanguageTranslations()` test fail. But IMHO the test must be adjusted, removing the code here is fine.

Comment (ausi) [reply core-bundle/src/Resources/contao/library/Contao/System.php:482]: Done in 6eef9c516bc06f885e60466e2dd2323cb7abbe4d

**Review by m-vo — DISMISSED** [review]

File: core-bundle/src/Resources/contao/library/Contao/Controller.php [hunk 2]
```diff
@@ -668,32 +668,6 @@ public static function getForm($varId, $strColumn='main', $blnModule=false)
...
-			$lang = substr($lang, 0, 2);
-
-			if (isset($GLOBALS['TL_LANG']['LNG'][$lang]))
-			{
-				$return[$lang] = $GLOBALS['TL_LANG']['LNG'][$lang] . '=' . $lang;
-			}
-		}
-
-		return '+' . implode(',', array_unique($return));
-	}
```
Comment (ausi) [inline core-bundle/src/Resources/contao/library/Contao/Controller.php:695]: @contao/developers Can I remove this method here? It is protected and not used anywhere.
I will add a deprecation for Contao 4.13 in a seperate PR.

Comment (ausi) [reply core-bundle/src/Resources/contao/library/Contao/Controller.php:695]: See #4567

**Review by Toflar — DISMISSED** [review]

**Review by leofeyer — DISMISSED** [review]

**Comment by leofeyer** [comment]
Thank you @ausi.

**Review by leofeyer — APPROVED** [review]

**Pull request merged**